sdk/dapp-sdk/src/provider/open.ts

```diff
@@ -15,5 +15,5 @@ export async function open(): Promise<void> {
         throw new Error('No previous session found')
     }
 
-    openKernelUserUI(discovery.walletType, session.status.kernel.userUrl ?? '')
+    openKernelUserUI(discovery.walletType, session.kernel.userUrl ?? '')
```

Comment:
we should ensure `additionalProperties: false` by default for our OpenRPC models... then we'd catch this at compile-time